### ISSUE TITLE
Epic - Event Inventory Management

### ISSUE THREAD

**Issue by zmelamed** [big-neon/docs#19]
Each Event can have multiple Ticket Allocations (aka ticket "types").

**Ticket Allocations**
Each Ticket allocation might have the following attributes:

- Name (ie. "General Admission", "VIP", "GA Early Bird")
- Description
- Quantity Available
- Quantity Reserved
- Quantity Sold
- Quantity Used
- Price
- On Sale Date/Time
- Sale End Date/Time

**URS**

- An Organization Admin and Organization User should be able to create Ticket Allocations for an event
- An Organization Admin and Organization User to put tickets on hold for an event
- An Organization Admin and Organization User to update ticket allocations for an event

**Comment by zmelamed** [comment]
@Krakaw @Timo614 @mikethetike please see the updated spec above explaining creation and redemption of holds, discount codes, and access codes.

**Comment by Timo614** [comment]
@zmelamed 

> An access redemption code is associated with 1 to many ticket types for a specific event and must be unique within the context of that event, if an access code is active and associated with a ticket allocation, that ticket allocation will only be visible if that access code is supplied by the user.

So the access code gates visibility for a given ticket type right? So a ticket type with any associated access will not be visible without it being present in the calls to the event ticket type data etc? And it needs to be provided in order to checkout tickets from it etc.

**Comment by zmelamed** [comment]
Yes, ticket_type==ticket allocation, and access codes act as a key/gate to see and purchase a ticket_type associated with that code
